custom_components/ha-glocaltokens/sensor.py

```diff
@@ -43,3 +62,92 @@ def icon(self):
     def device_class(self):
         """Return de device class of the sensor."""
         return "glocaltokens__custom_device_class"
+
+class GlocaltokensAlarmSensor(GlocaltokensEntity):
+    """Representation of a Sensor."""
+
+    def __init__(self, coordinator, entry, device):
+        """Initialize the sensor."""
+        super().__init__(coordinator, entry)
+        self._name = device[GLOCALTOKENS_DEVICE_NAME]
+        self._token = device[GLOCALTOKENS_TOKEN]
```

Comment:
If we keep the token in the `GlocaltokensAlarmSensor` do we need the separate `GlocaltokensSensor` sensor holding the same token? We should probably reuse it

Reply:
No, i just forgot to removed from timer and alarm sensor

Reply:
We probably should do that to make it easier in the future to add new things